### PR TITLE
feat: add action "Create async replication"

### LINKED ISSUE #783: Add Create Async Replication action to db context menu
Database context menu accessible under ... control must have an Item `Create Async Replication` that fills Query Editor with template query

```
CREATE ASYNC REPLICATION my_replication
FOR <remote_table_name> AS <local_table_name> [, <remote_table_name> AS <local_table_name> ...]
WITH (
  ENDPOINT=<endpoint>, -- endpoint, e.g. grpcs://mydb.ydb.tech:2135/
  DATABASE=<path_to_database>, --/cluster/dynamo
  USER=<user>,
  PASSWORD=<pwd>,
  ...
);
```

### PR DESCRIPTION
**Pull request by Raubzeug**
closes #783

### CODE REVIEW

File: src/containers/Tenant/utils/schemaActions.ts [hunk 1]
```diff
@@ -49,6 +50,7 @@ const bindActions = (
     return {
         createTable: inputQuery(createTableTemplate, 'script'),
         createColumnTable: inputQuery(createColumnTableTemplate, 'script'),
+        createAsyncReplication: inputQuery(createAsyncReplicationTemplate, 'script'),
```
Comment (ValeraS) [inline src/containers/Tenant/utils/schemaActions.ts:53]: What about stop replication https://github.com/ydb-platform/ydb-embedded-ui/issues/783#issuecomment-2077381564

Comment (Raubzeug) [reply src/containers/Tenant/utils/schemaActions.ts:55]: Added

File: src/containers/Tenant/utils/queryTemplates.ts [hunk 2]
```diff
@@ -42,6 +42,17 @@ CREATE TABLE \`${path}/ydb_column_table\` (
...
+    return `CREATE ASYNC REPLICATION my_replication
+FOR <remote_table_name> AS <local_table_name> [, <remote_table_name> AS <local_table_name> ...]
+WITH (
+    ENDPOINT=<endpoint>, -- endpoint, e.g. grpcs://mydb.ydb.tech:2135/
+    DATABASE=<path_to_database>, --/cluster/dynamo
+    USER=<user>,
+    PASSWORD=<pwd>,
+    ...
+);`;
+};
```
Comment (artemmufazalov) [inline src/containers/Tenant/utils/queryTemplates.ts:55]: I don't really like `<>` in this template. In other template we use some preset values, so you can alter them and don't need to think of the form. For example - should I use some marks and which marks should I use - ', " or `. For me, it's better to have the form like:
```
WITH (
    ENDPOINT="grpcs://mydb.ydb.tech:2135/",
    DATABASE="your_database",
    USER="user",
    PASSWORD="your_password"
)
```

Comment (Raubzeug) [reply src/containers/Tenant/utils/queryTemplates.ts:55]: agree with you!

File: src/containers/Tenant/utils/queryTemplates.ts [hunk 3]
```diff
@@ -42,6 +42,17 @@ CREATE TABLE \`${path}/ydb_column_table\` (
...
 };
+export const createAsyncReplicationTemplate = () => {
+    return `CREATE ASYNC REPLICATION my_replication
+FOR <remote_table_name> AS <local_table_name> [, <remote_table_name> AS <local_table_name> ...]
+WITH (
+    ENDPOINT=<endpoint>, -- endpoint, e.g. grpcs://mydb.ydb.tech:2135/
+    DATABASE=<path_to_database>, --/cluster/dynamo
+    USER=<user>,
+    PASSWORD=<pwd>,
+    ...
```
[the remark this answers is not in the document]
Comment (Raubzeug) [reply src/containers/Tenant/utils/queryTemplates.ts:57]: @CyberROFL said, that we should not expose this option in template

File: src/containers/Tenant/utils/queryTemplates.ts [hunk 4]
```diff
@@ -42,6 +42,21 @@ CREATE TABLE \`${path}/ydb_column_table\` (
...
+CREATE ASYNC REPLICATION my_replication
+FOR \`/remote_database/table_name\` AS "local_table_name" --[, \`/remote_database/another_table_name\` AS "another_local_table_name" ...]
+WITH (
+    CONNECTION_STRING=\`grpcs://mydb.ydb.tech:2135/?database=/remote_database\`,
+    TOKEN_SECRET_NAME = "secret_name",
+    -- ENDPOINT="mydb.ydb.tech:2135",
+    -- DATABASE=\`/remote_database\`,
+    -- USER="user",
+    -- PASSWORD_SECRET_NAME="your_password"
+    ...
```
Comment (artemmufazalov) [inline src/containers/Tenant/utils/queryTemplates.ts:57]: Please, remove redundant `...`

**Review by artemmufazalov — CHANGES_REQUESTED** [review]

**Review by artemmufazalov — APPROVED** [review]

**Pull request merged**